README.md

```diff
@@ -1,5 +1,6 @@
 # backend-application
 [![Build Status](https://travis-ci.org/racketometer/backend-application.svg?branch=master)](https://travis-ci.org/racketometer/backend-application)
+[![Coverage Status](https://coveralls.io/repos/karnich/backend-application/badge.svg?branch=master)](https://coveralls.io/r/karnich/backend-application?branch=master)
```

Comment:
I expected this to be: `https://coveralls.io/github/racketometer/backend-application` instead of `karnich`?